### PR TITLE
Fix CI Fails

### PR DESCRIPTION
**Pull request by FG-TUM**
# Description

The upgrade of the dockerfiles to ubuntu 20 broke some stuff (of course....)

- [x] Remove duplicated doc in `Math.h`
- [x] Recreate a non OpenMP Jenkins job
- [x] Fixed call to start `sph-main-mpi`

## Resolved Issues

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

- [x] Jenkins
  - [x] waiting for new docker images

### CODE REVIEW

File: src/autopas/utils/Math.h [hunk 1]
```diff
@@ -154,13 +140,6 @@ T safeMul(const T &a, const T &b, const T &valUnderflow = std::numeric_limits<T>
  * If over or underflow is detected, the function returns the specified values.
  *
  * @note Underflow here refers to a value more negative than representable and not the underflow gap around zero.
- *
- * @tparam T
- * @param a
- * @param b
- * @param valUnderflow Return value in case of underflow.
- * @param valOverflow Returns value in case of overflow.
- * @return Product or valUnderflow or valOverflow.
```
[the remark this answers is not in the document]
Comment (SteffenSeckler) [reply src/autopas/utils/Math.h:163]: ok, seems like a doxygen bug, similar but not identical to: https://github.com/doxygen/doxygen/issues/7700

the second workaround from https://github.com/doxygen/doxygen/issues/7700#issuecomment-870556809
should probably be applied, i.e., renaming the template arguments to TF (for floats) or TI (for integrals)...

[the remark this answers is not in the document]
Comment (SteffenSeckler) [reply src/autopas/utils/Math.h:163]: this file could also include an `@todo c++20 replace with std::integral / std::floating_point.`

[the remark this answers is not in the document]
Comment (FG-TUM) [reply src/autopas/utils/Math.h:163]: There are two template versions for all of these functions, one for float and one for integral. Doxygen treats them as one, hence the duplication

[the remark this answers is not in the document]
Comment (SteffenSeckler) [reply src/autopas/utils/Math.h:163]: you can fix that duplication by naming the template parameters differently, e.g., TF and TI (see above)

[the remark this answers is not in the document]
Comment (FG-TUM) [reply src/autopas/utils/Math.h]: fixed via a6a44bb

**Review by SteffenSeckler — APPROVED** [review]

**Pull request merged**